source/views/sis/course-search/detail/index.js

```diff
@@ -82,28 +97,38 @@ function Information({course}: {course: CourseType}) {
 }
 
 function Schedule({course}: {course: CourseType}) {
-	if (!course.times) {
+	if (!course.offerings) {
 		return null
 	}
-	const times = convertTimeStringsToOfferings(
-		{
-			times: course.times,
-			locations: course.locations,
-		},
-		{groupBy: 'day'},
-	)
-	const schedule = times.map(time => {
-		const hours = time.times.map(time => {
-			const start = moment.tz(time.start, 'hmm', CENTRAL_TZ).format('h:mm A')
-			const end = moment.tz(time.end, 'hmm', CENTRAL_TZ).format('h:mm A')
+	const groupedByDay = groupBy(course.offerings, 'day')
```

Comment:
Please use the function second argument instead of the key shorthand, for consistency with the rest of the code – `groupBy(course.offerings, c => c.day)`